integrations/optimizely/test/index.test.js

```diff
@@ -1253,23 +1544,56 @@ describe('Optimizely', function() {
       });
     });
   });
-});
 
-/*
- * execute AsyncTest
- *
- * Prevent tests from hanging if deepEqual fails inside `tick`
- * @api private
- * @param {Function} done cb
- * @param {Function} function that runs test
- */
-function executeAsyncTest(done, test) {
-  tick(function() {
-    try {
-      test();
-      done();
-    } catch (e) {
-      done(e);
-    }
+  describe('#initialize on Web', function() {
```

Comment:
A little confusing to tell in the diff, but I moved down both `#initialize with Web` and `#initialize with Edge` to the end of the test suite. `#initialize with Edge` causes `#sendWebDecisionToSegment` tests to completely fail, but I knew that tests were passing individually. This leads me to believe that there is _something_ (unsure if it already existed or if I introduced it) in the testing module where variables/data aren't being reset before each test. 

Putting all the test to the bottom passes all of them now. Maybe someone on the Segment side may have an idea why this is happening?

Reply:
I'm happy to help look into this too, if that's helpful. I might have some context from my PR.